homework/index.html

```diff
@@ -3,8 +3,9 @@
 <head>
   <meta charset="UTF-8">
   <title> Homework - Note List </title>
-  <script defer src="list.js"></script>
-  <script defer src="content.js"></script>
+  <script src="list.js"></script>
```

Comment:
To set `defer` is to prevent loading JS blocks HTML loading, so you should keep it.